lib/ansible/plugins/filter/mathstuff.py

```diff
@@ -86,6 +86,23 @@ def _do_fail(e):
     return c
 
 
+@environmentfilter
+def duplicate(environment, a):
+    if isinstance(a, Hashable):
```

Comment:
This `Hashable` check isn't needed.

Additionally, instead of the current code, make use of `collections.Counter`


```
from collections import Counter
@environmentfilter
def duplicate(environment, a):
    return [k for k, v in Counter(a).items() if v > 1]
```